Gateway/Command/PayByMailCommand.php

```diff
@@ -163,20 +153,20 @@ protected function getFormFields($payment, $paymentAmount = false)
         $customerId = $order->getCustomerId();
 
         // get locale from store
-        $shopperLocale = (!empty($shopperLocale)) ? $shopperLocale : $this->_adyenHelper->getStoreLocale($storeId);
-        $countryCode = (!empty($countryCode)) ? $countryCode : false;
+        $shopperLocale = $this->_adyenHelper->getStoreLocale($storeId);
 
         // if directory lookup is enabled use the billingadress as countrycode
-        if ($countryCode == false) {
-            if (is_object($order->getBillingAddress()) && $order->getBillingAddress()->getCountryId() != "") {
-                $countryCode = $order->getBillingAddress()->getCountryId();
-            } else {
-                $countryCode = "";
-            }
+        if (is_object($order->getBillingAddress()) && $order->getBillingAddress()->getCountryId() != "") {
+            $countryCode = $order->getBillingAddress()->getCountryId();
+        } else {
+            $countryCode = "";
         }
 
-        $deliveryDays = $this->_adyenHelper->getAdyenHppConfigData('delivery_days', $storeId);
```

Comment:
Same here? The config was removed?

Reply:
@acampos1916 Indeed the same as above, it did not exist anymore since the 4.x.x (not sure about the exact version maybe 4.2.0).